products/joomla.md

```diff
@@ -14,18 +14,28 @@ releaseDateColumn: true
 auto:
 -   git: https://github.com/joomla/joomla-cms.git
 
+# eol see https://developer.joomla.org/roadmap.html
 releases:
--   releaseCycle: "4"
-    eol: false
+-   releaseCycle: "5"
+    releaseDate: 2023-10-14
     support: true
+    eol: false
```

Comment:
```suggestion
    eol: 2027-20-19
```

According to https://developer.joomla.org/roadmap.html.

Reply:
Good catch :+1: